docs/usage.md

```diff
@@ -0,0 +1,500 @@
+# Usage
+
+:warning: This file shows example commands for reading, creating, updating and deleting redmine components. As this are only examples, please **make sure, that you do not execute them at your production Redmine instance and may destroy real life content!**
+
+## Installation
+
+### With Composer
+
+Please make sure you have install `php-remine-api` with an PSR-4 autoloader like [Composer](https://getcomposer.org/).
+
+```bash
+php composer.phar require kbsali/php-redmine-api
+```
+
+This will create a new entry in your `composer.json`.
+
+```diff
+{
+    "require": {
++        "kbsali/php-redmine-api": "^1.6"
+    }
+}
+
+```
+
+### ... or without Composer
+
+As an alternative you can download the code with all required dependencies from [php-download.com](https://php-download.com/package/kbsali/redmine-api) and unzip all files in your project folder.
+
+### Start your project
+
+Create your project e.g. in the `index.php` by require the `vendor/autoload.php`
+
+```diff
++<?php
++
++require_once 'vendor/autoload.php';
+```
+
+### Instantiate a Redmine Client
+
+You can choose between the navite curl client without other PHP dependencies or the PSR-18 compatible client.
+
+#### Native curl Client `Redmine\Client`
+
+Every Client requires a URL to your Redmine instance and either an Apikey...
+
+```diff
+<?php
+
+require_once 'vendor/autoload.php';
++
++// Instantiate with ApiKey
++$client = new Redmine\Client('http://localhost', '1234567890abcdfgh');
+```
+
+... or valid username/password.
+
+
+```diff
+<?php
+
+require_once 'vendor/autoload.php';
++
++// Instantiate with ApiKey with Username/Password
++$client = new Redmine\Client('http://redmine.example.com', 'username', 'password');
+```
+
+> :bulb: For security reason it is recommended that you use a ApiKey instead of username/password.
```

Comment:
```suggestion
> :bulb: For security reason it is recommended that you use an ApiKey rather than your username/password.
```

Reply:
Thank you very much. :heart:  I'm not a native speaker.